apollo.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  client: {
+    service: {
+      name: 'kukkuu',
+      url: 'http://localhost:8001/graphql',
```

Comment:
config.js ? -> .ts instead?

Reply:
Required adding `export {}` to the file to address ts complaint: 

> Cannot compile namespaces when the '--isolatedModules' flag is provided